vms/platformvm/validators/manager_test.go

```diff
@@ -0,0 +1,445 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package validators
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/chains"
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/choices"
+	"github.com/ava-labs/avalanchego/snow/uptime"
+	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/timer/mockable"
+	"github.com/ava-labs/avalanchego/utils/units"
+	"github.com/ava-labs/avalanchego/vms/platformvm/blocks"
+	"github.com/ava-labs/avalanchego/vms/platformvm/config"
+	"github.com/ava-labs/avalanchego/vms/platformvm/metrics"
+	"github.com/ava-labs/avalanchego/vms/platformvm/reward"
+	"github.com/ava-labs/avalanchego/vms/platformvm/state"
+)
+
+// AVAX asset ID in tests
+var defaultRewardConfig = reward.Config{
+	MaxConsumptionRate: .12 * reward.PercentDenominator,
+	MinConsumptionRate: .10 * reward.PercentDenominator,
+	MintingPeriod:      365 * 24 * time.Hour,
+	SupplyCap:          720 * units.MegaAvax,
+}
+
+func TestVM_GetValidatorSet(t *testing.T) {
+	ctrl := gomock.NewController(t)
```

Comment:
should the mock controller be instantiated per test (near the definition of require)?

Reply:
Done